bang/magicollections.py

```diff
@@ -900,20 +900,42 @@ class ListView(cls.ListView):
     'korean_start_date': 'date', 'korean_end_date': 'date',
     'type': 'toggler',
 }
+EVENT_CUTEFORM = {
```

Comment:
Copy-pasting cuteform details might be a problem in the future if we decide to change something and we forget to change it at both places!

What you can do is maybe have a `EVENT_CUTEFORM_BASE = ....` for all shared stuff, then for each you can do:

```
EVENT_CUTEFORM_ADD_EDIT = EVENT_CUTEFORM_BASE.copy() # make a copy of the base
EVENT_CUTEFORM_ADD_EDIT['main_card'] = { ... } # add settings that are only for the form
```

Reply:
Technically it is the list and item views that get extra settings, so I'll make the add/edit a base instead